recipes/schema/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2012 Vladimir Keleshev, <vladimir@keleshev.com>
```

Comment:
This LICENSE file can be removed. It is included in the pypi tarball and named `LICENSE-MIT`